packages/next-swc/crates/next-dev-tests/tests/integration/next/app/404-custom/input/app/test.tsx

```diff
@@ -49,7 +49,8 @@ function runTests(harness: Harness, iframe: HTMLIFrameElement) {
   )
 
   // TODO: This test is flaky, so it needs a particularly long timeout.
-  it(
+  // TODO(WEB-980) Fix this test once we no longer throw an error when rendering a 404 page.
```

Comment:
Checked with @alexkirsz that not-found is not handled well in turbopack so we add a ticket to track later